src/Routes/Devices/Inventory.js

```diff
@@ -50,6 +50,71 @@ const Inventory = () => {
     setIsRowSelected(isRow);
   };
 
+  const setCanBeUpdated = () => {
+    let canBeUpdated = false;
+    if (checkedDeviceIds.length > 0) {
+      let initialImage = checkedDeviceIds[0].imageSetId;
+      for (let device of checkedDeviceIds) {
+        if (device.imageSetId !== initialImage) {
+          canBeUpdated = false;
+          break;
+        }
+        if (!canBeUpdated && device.updateImageData) {
+          canBeUpdated = true;
+        }
+      }
+    }
+    return canBeUpdated;
+  };
+
+  /*
+  const setCanBeUpdated = () => {
+    let canBeUpdated = false;
+    if (checkedDeviceIds.length > 0) {
+      canBeUpdated = 
+        checkedDeviceIds.reduce(checkImageAndUpdateStatus, {
+          previousId: null,
+          buttonEnabled: false,
+        })
+      ).buttonEnabled;
+      //console.log(canBeUpdated);
+    }
+    return canBeUpdated;
+  };
+
+  const checkImageAndUpdateStatus = (previousDevice, currentDevice) => {
+    console.log(currentDevice.updateImageData);
+    console.log(previousDevice);
+    if (
+      previousDevice.previousId &&
+      previousDevice.previousId !== currentDevice.ImageSetId
+    ) {
+      previousDevice.buttonEnabled = false;
+      return previousDevice;
+    }
+
+    previousDevice.previousId = currentDevice.imageSetId;
+    if (previousDevice.buttonEnabled === false) {
+      previousDevice.buttonEnabled = currentDevice.updateImageData;
+    }
+
+    console.log(previousDevice?.buttonEnabled);
+    console.log(previousDevice);
+    return previousDevice;
+  };
+  */
```

Comment:
would be great to remove this commented code

Reply:
removed!